Add all the mobile app (beta) documentation (#21110)
opf/openproject: docs/mobile-app-guide/README.md

---
sidebar_navigation:
  title: Mobile app guide (Beta)
  priority: 972
description: User guide for the OpenProject Mobile application.

> keywords: Mobile app guide

Reviewer: ```suggestion
keywords: Mobile app guide, mobile, mobile application, iOS, android, app store, appstore, app-store, google play
```

---
# Mobile app guide (Beta)

> We are excited to announce the **Beta Release** of the **OpenProject Mobile app** — now available in the **App Store** and **Google Play** Store. This Beta version marks the beginning of our journey to bring **OpenProject** to your mobile devices, enabling you to stay connected and productive wherever you are.

Reviewer: ```suggestion
We are excited to announce the **Beta Release** of the **OpenProject Mobile app** — now available in the **App Store** and **Google Play Store**. This Beta version marks the beginning of our journey to bring **OpenProject** to your mobile devices, enabling you to stay connected and productive wherever you are.
```

> [!NOTE]

> > The app is currently under active development, and core features are being continuously improved. We invite you to test the app, share feedback, and help shape the future of OpenProject Mobile.

Reviewer: Maybe we could also add a hint that both the app, AND this document are a work in progress? Something like: 

Hence, please also keep in mind that his page is a work in progress and will be updated along with the mobile app improvements. 
Last edited on: date, 2025.

Author: Hi @MayaBerd aren't all documentations living elements? Do we do that for the web app documentation anywhere?

Reviewer: @marcalcobe True. I've recently added this to the overview of the JIRA migration docs, to try and keep any questions on our own tool at bay.

*   **Local notifications:** Stay up to date with mentions, comments, and task updates through local notifications — even when the app isn’t open. *These notifications are not yet real-time push notifications.*
*   **Create work packages fast:** Quickly add new work packages in a distraction-free interface designed for fast, mobile-friendly input.
*   **Run timers in focus mode:** Track your work time effortlessly with built-in timers that run in the background, helping you stay focused and accurate with time logs.
*   **Configure modules and dashboards:** Customize what you see by enabling only the modules and views most relevant to you — keeping your mobile workspace clean and efficient.
*   **Optimized touch interface:** Navigate with ease using gestures, adaptive layouts, and mobile-first design tailored for smaller screens.

These mobile-specific capabilities enhance your ability to **work efficiently, document instantly, and stay connected** — all while maintaining focus and flexibility on the go.

## Coming Soon

The app is still under development, and many core features are planned for future updates, including:

> *   **Deep-linking (including on-premises support):** Seamlessly open specific work packages, projects, or comments directly from links in emails, chats, or browser pages — whether you’re using the cloud or an on-premises instance.

Reviewer: Is there already a framework for support options? Is it tied to the app support levels? We should clarify this somewhere.

Author: No, there are no supports levels related to the mobile app and for now there is no intention of making anything related to the app Enterprise as far as I know.

I don't understand what do we need to clarify here. Deep-linking has nothing to do with our support plans. Maybe we should remove the "support" from the sentence "on-premises support" to avoid confusion.

*   **Multi-device UI:** Enjoy a consistent, responsive experience across phones, tablets and macOS, with layouts optimized for each device size and orientation.
*   **Real-time push notifications:** Receive updates instantly as they happen — from mentions and comments to task status changes — ensuring you never miss important activity.
*   **Write internal comments:** Add internal comments directly from the app, enabling secure collaboration within your project team while keeping external communications separate.
*   **Meeting agendas in the app:** Access and review meeting agendas on the go to stay prepared and aligned with your team wherever you are.

These upcoming features will make the OpenProject Mobile App even more connected, collaborative, and aligned with the full OpenProject experience.

## Feedback and Involvement

As a Beta user, your feedback is invaluable. If you encounter issues or have suggestions, please let us know:&nbsp;

*   Using the **flow to provide feedback** directly from the **app** in the user settings.

> ![Entry point for the mobile app flow to provide feedback](mobile_app_feedback_flow_entrypoint.png)

Reviewer: ```suggestion
![Entry point for the OpenProject mobile app flow to provide user feedback](mobile_app_feedback_flow_entrypoint.png)
```

> *   Create a **feedback work package** directly in OpenProject **Community**.

Reviewer: ```suggestion
*   Create a **feedback work package** directly in [OpenProject **Community** instance](https://community.openproject.org/) and assign it to *Stream Mobile App*.
```

Your input directly helps improve the mobile experience and ensure a stable, feature-rich public release.

## Availability

The **OpenProject Mobile App (Beta)** is available now for:

*   **iOS:** [App Store link](?).
*   **Android:** [Google Play link](?).

## Requirements

To access and use the **OpenProject Mobile App (Beta)**, you’ll need the following:

*   **An active OpenProject account:** Either from an **OpenProject Cloud** workspace or an **OpenProject On-premises** installation with API access enabled.

> *   Having a **signed certificate** (**https** not http) on your instance to be able to log in.

Reviewer: ```suggestion
*   Having a **signed certificate** (**https**, not http) on your instance to be able to log in.
```

> *   **Minimum OpenProject version:** 17.0.0

Reviewer: ```suggestion
*   **OpenProject version:** 17.0.0 or above
```

> [!NOTE]
> If you have a previous version of OpenProject you can connect your OpenProject instance by asking your administrator to enable the Built in OAuth applications flag under_ `_{BASE_URL}/admin/settings/experimental_`.
*   **Minimum system requirements:**
    *   **iOS 15** or later
    *   **Android 12** or later
*   **Built-in OAuth applications enabled:** Make sure that the built-in OAuth applications are **enabled in your administration settings** (`{BASE_URL}/admin/oauth/applications`). This is required for successful login from the mobile app.
![Applications setting to enable the built-in OAuth](mobile_app_oauth_authentication.png)
*   **Network connection:** Internet access is required for syncing data with your OpenProject instance.
> [!NOTE]

> > Some features, such as deep-linking and realtime push notifications, may depend on your organization’s configuration or will become available in future updates.

Reviewer: ```suggestion
> Some features, such as deep-linking and real-time push notifications, may depend on your organization’s configuration or will become available in future updates.
```

## Disclaimer

> > [!NOTE]

Reviewer: ```suggestion
> [!IMPORTANT]
```
